medusa/show_updater.py

```diff
@@ -107,8 +113,8 @@ def run(self, force=False):
 
                         if isinstance(error, HTTPError):
                             if error.response.status_code == 503:
-                                logger.warning(u'API Service offline: '
-                                               u'This service is temporarily offline, try again later.')
+                                logger.info(u'API Service offline: '
```

Comment:
@bobbysteel May I ask why this was changed?

Reply:
oh sorry you mean this case of warning to info? happy to revert - given the unreliability of theTVDB I think warning is a bit much. philosophically, i feel itinerant connection issues don't warrant WARNING labels but please feel free to change back if you prefer.

Reply:
Yes, this specific case. Because it doesn't fit in with the other changes.
I don't think I ever got that warning.
I think server errors are caught by the `IndexerUnavailable` exception.
I'll revert it, just in case.

Reply:
Okey doke. Sorry for adding it in - think I was just demoting a few things and assumed this was triggered as well in the tvdb case.